locator/load_sketch.hh

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#pragma once
+
+#include "locator/topology.hh"
+#include "locator/token_metadata.hh"
+#include "locator/tablets.hh"
+
+#include <seastar/core/smp.hh>
+
+#include <optional>
+#include <vector>
+
+namespace locator {
+
+/// A data structure which keeps track of load associated with data ownership
+/// on shards of the whole cluster.
+class load_sketch {
+    using shard_id = seastar::shard_id;
+    struct shard_load {
+        shard_id id;
+        size_t load;
```

Comment:
What are the units? tablets or bytes? If bytes, should be uint64_t since size_t is local RAM sized.

Reply:
>What are the units?

tablets.